src/pagination/pagination.ts

```diff
@@ -111,6 +112,16 @@ export class NgbPaginationPrevious {
   constructor(public templateRef: TemplateRef<NgbPaginationLinkContext>) {}
 }
 
+/**
+ * A directive to match the 'pages' whole content
+ *
+ * @since 8.1.0
+ */
+@Directive({selector: 'ng-template[ngbPaginationPages]'})
+export class NgbPaginationPages {
+  constructor(public templateRef: TemplateRef<NgbPaginationLinkContext>) {}
```

Comment:
I think it merits a separate context like `NgbPaginationPagesContext` even though it might be a copy-paste of the link one at the moment.